apps/labs/public/posts/introducing-intern-cohort-2022/hero.svg

```diff
@@ -0,0 +1 @@
43459 315.0373c1.9248352 4.5860596 8.435745 4.5161743 10.286011 -0.07455444c1.7756958 -4.6606445 -2.8849335 -9.172119 -7.4010773 -7.177368c-2.8103638 1.2583618 -4.1433105 4.4415894 -2.8849335 7.2519226z" fill-rule="evenodd"/><path stroke="#452391" stroke-width="1.5826771653543308" stroke-miterlimit="15.875" stroke-linecap="butt" d="m229.43459 315.0373c1.9248352 4.5860596 8.435745 4.5161743 10.286011 -0.07455444c1.7756958 -4.6606445 -2.8849335 -9.172119 -7.4010773 -7.177368c-2.8103638 1.2583618 -4.1433105 4.4415894 -2.8849335 7.2519226z" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m165.43459 187.03731c1.9248352 4.5860596 8.435745 4.516144 10.286011 -0.0745697c1.7756958 -4.6606293 -2.8849335 -9.172119 -7.4010773 -7.177368c-2.8103638 1.2583618 -4.1433105 4.441574 -2.8849335 7.251938z" fill-rule="evenodd"/><path stroke="#973896" stroke-width="1.5826771653543308" stroke-miterlimit="15.875" stroke-linecap="butt" d="m165.43459 187.03731c1.9248352 4.5860596 8.435745 4.516144 10.286011 -0.0745697c1.7756958 -4.6606293 -2.8849335 -9.172119 -7.4010773 -7.177368c-2.8103638 1.2583618 -4.1433105 4.441574 -2.8849335 7.251938z" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m143.86717 479.39386c-6.660034 2.8849182 -13.389984 -3.700531 -10.72876 -10.360565c2.6658783 -6.72995 12.061707 -6.8790894 14.94664 -0.21905518c1.7011261 4.0687256 -0.14915466 8.803925 -4.2178802 10.57962z" fill-rule="evenodd"/><path stroke="#452391" stroke-width="1.5826771653543308" stroke-miterlimit="15.875" stroke-linecap="butt" d="m143.86717 479.39386c-6.660034 2.8849182 -13.389984 -3.700531 -10.72876 -10.360565c2.6658783 -6.72995 12.061707 -6.8790894 14.94664 -0.21905518c1.7011261 4.0687256 -0.14915466 8.803925 -4.2178802 10.57962z" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m159.86717 327.39386c-6.660034 2.8849182 -13.389984 -3.700531 -10.72876 -10.360565c2.6658783 -6.72995 12.061707 -6.8790894 14.94664 -0.21905518c1.7011261 4.0687256 -0.14915466 8.803925 -4.2178802 10.57962z" fill-rule="evenodd"/><path stroke="#452391" stroke-width="1.5826771653543308" stroke-miterlimit="15.875" stroke-linecap="butt" d="m159.86717 327.39386c-6.660034 2.8849182 -13.389984 -3.700531 -10.72876 -10.360565c2.6658783 -6.72995 12.061707 -6.8790894 14.94664 -0.21905518c1.7011261 4.0687256 -0.14915466 8.803925 -4.2178802 10.57962z" fill-rule="evenodd"/><path fill="#000000" fill-opacity="0.0" d="m255.86717 207.39386c-6.660034 2.8849335 -13.389984 -3.700531 -10.72876 -10.36058c2.6658783 -6.72995 12.061691 -6.8790894 14.94664 -0.21903992c1.7011108 4.0687256 -0.1491394 8.803925 -4.2178802 10.57962z" fill-rule="evenodd"/><path stroke="#452391" stroke-width="1.5826771653543308" stroke-miterlimit="15.875" stroke-linecap="butt" d="m255.86717 207.39386c-6.660034 2.8849335 -13.389984 -3.700531 -10.72876 -10.36058c2.6658783 -6.72995 12.061691 -6.8790894 14.94664 -0.21903992c1.7011108 4.0687256 -0.1491394 8.803925 -4.2178802 10.57962z" fill-rule="evenodd"/></g></svg>
```

Comment:
Is this file being used at all?

Reply:
Not currently. I added it in case the hero I made doesn't look good, and I can switch to a generic one quick and make my hero an in blog image.

Reply:
Ah okay, could I ask you to remove it if you end up not using it, before merging this PR?

Reply:
Done.